### DISCUSSION TITLE
NeXTSpirit icons ?

### DISCUSSION THREAD

**Discussion by galgot** [trunkmaster/nextspace#385] [Ideas]
Hi,
I’ve been following this project for a while, and I’m impressed. 
I don’t have much in term of coding knowledge (nothing) to contribute to it.
Also have no experience with Cent-OS, as I usually install Debian.
But still, would you be interested in using this ? :

[NextSpiritForGnustep.tar.zip](https://github.com/trunkmaster/nextspace/files/3311860/NextSpiritForGnustep.tar.zip)

Its an icon set I’ve done originally for Mac OS X (big 1024x1024 icons), completely inspired by original NeXTstep icons, just higher res.
http://galgot.free.fr/wordpress/?p=1380
http://galgot.free.fr/wordpress/?p=1410
I’ve done these smaller 48x48 for Riccardo Mottola of the GNUstep project for the "Heritage" theme.
Can be used for NextSpace too, my pleasure.

Best regards,

**Comment by trunkmaster** [comment]
Hi,
I've already visited your site and very impressed with your high-res icons. I've even tried to resize some of them some time ago but without much success. I've got blurred icons, loosing much of shiny details from your original images. Finally I've dropped the idea for the future and returned to coding. :)

Actually, I badly need someone with taste like yours. I think much of NeXTSTEP attractiveness is due to the excellent sharp and expressive graphics. I'm sure we can create together new desktop with refined icons while retaining the NeXT idea/spirit.

Are you ready to spend some time listening to my critics, suggestions and making some changes? ;)

**Comment by galgot** [comment]
Well if you want some small changes on these existing icons, yes :)
As for redoing an all set , I’m afraid I will not have time for that as I’m committed to other works for the moment...
But, sure , tell me. And I'll see what I can do.

**Comment by armm77** [comment]
Hi, @trunkmaster and @galgot , I suggest that Mr. @galgot create a logo for the desk. I feel like a little life was lost after removing the proprietary NeXT logo. :)
![Screen Shot 2019-06-20 at 7 34 16 PM](https://user-images.githubusercontent.com/3104815/59889571-97097b00-9392-11e9-9115-549ee1c90ac5.png)

**Comment by galgot** [comment]
Hi, I tried something like this 

![NextSpace-Mockup01](https://user-images.githubusercontent.com/12213521/60341950-6dfb7280-99b0-11e9-991d-55fcc1d90b77.jpg)

![NextSpace-Mockup02](https://user-images.githubusercontent.com/12213521/60341960-7489ea00-99b0-11e9-909e-022e4a7eaa43.jpg)

![NextSpace-Mockup03](https://user-images.githubusercontent.com/12213521/60342066-c599de00-99b0-11e9-952a-5b6fb36b2fb0.jpg)

**Comment by galgot** [comment]
Noticed I closed the Issue by mistake last time, sorry bout that. I'm not very familiar with GitHub.

**Comment by trunkmaster** [comment]
@galgot, it's amazing! Fantastic idea! Well done! 
Could you please make black part of image darker and drop shadow like in NeXT image?

BTW, what is gamma correction setting do you use? My Xorg gamma setting is closer to NeXT:
```
[me@localhost]~% xrandr --verbose | grep Gamma
        Gamma:      0.80:0.80:0.80
```

**Comment by galgot** [comment]
This works ?
![NextSpace-Mockup01-V2](https://user-images.githubusercontent.com/12213521/60430919-c32cc380-9bfe-11e9-8312-d4bdc38efed7.jpg)

![NextSpace-Mockup03-V2](https://user-images.githubusercontent.com/12213521/60430942-d50e6680-9bfe-11e9-9cd0-aa7e55f39fe0.jpg)

Note, if I do it too dark, you can't see the 3d effect of the two levels , and with this shadow you have more difficulty reading a "S" .

**Comment by trunkmaster** [comment]
Much better. Could you please share source files (.blend, .xcf)?

**Comment by galgot** [comment]
Sure, here is the file, with layers passes, V.1 and V.2 as layers too. Gimp-2.10.
At the resolution which I rendered it, 2000x2000.
[NextSpace-Logo-2000x2000.xcf.zip](https://github.com/trunkmaster/nextspace/files/3346696/NextSpace-Logo-2000x2000.xcf.zip)

**Comment by trunkmaster** [comment]
Thank you. I'll play with it tomorrow - my GIMP-2.8 won't open this file.

**Comment by trunkmaster** [comment]
@galgot, I've played a bit with images. My observations:
1. On low resolution "NEXT" fade to "S" edges letters "e", "N" and "T". Try to move text out of edges visually (text shift or camera POV).
2. I guess you use light from top right to create shadow of text. I think right side if image needs more contrast against background. Maybe "S" at right side should be less reflective? What do you think?

Here what I've got with Dock icon:
![Dock0](https://user-images.githubusercontent.com/23114975/60501245-60056480-9cc4-11e9-81ac-8ac1324227ad.png)

and Login Panel
![panel](https://user-images.githubusercontent.com/23114975/60501092-1fa5e680-9cc4-11e9-91b5-60820c057eec.png)

Could you please share .blend file so I can play with POV?
BTW, Login Panel .xcf located inside ![Graphics](https://github.com/trunkmaster/nextspace/tree/master/Graphics) folder.

**Comment by galgot** [comment]
1. Sorry, Want me to move the letters down a bit to have their edges all within the black "S" ?
2. Light is directly from above. I can try moving light a bit left if you want.

Yes, looks good. but I think we should just discard the drop shadow altogether for the tile.
Original NeXT dock tile didn’t had it. It permit better centering on the tile and more clarity. think there should be 1 or 2 Pixels space between the tile edge and logo on top and bottom too.

Will try that later.

Sorry, I can contribute images, for using free as you like, that can be modified as you like.
But I never share 3d Models.

**Comment by trunkmaster** [comment]
>1. Sorry, Want me to move the letters down a bit to have their edges all within the black "S" ?
>
Yes.
>2. Light is directly from above. I can try moving light a bit left if you want.
> 
Try to make "S" less reflective so the right part of it will not mix with background.
Here is what I've got without drop shadow and increasing contrast by 10.
![Dock0](https://user-images.githubusercontent.com/23114975/60505396-51bb4680-9ccc-11e9-9233-bc5d197517fd.png)
Less details are visible on "S" surface but icon becomes more distinct against tile.
> 
> Yes, looks good. but I think we should just discard the drop shadow altogether for the tile.
> Original NeXT dock tile didn’t had it. It permit better centering on the tile and more clarity. think there should be 1 or 2 Pixels space between the tile edge and logo on top and bottom too.
> 
You're right.
> Will try that later.
> 
> Sorry, I can contribute images, for using free as you like, that can be modified as you like.
> But I never share 3d Models.
>
No problem.

**Comment by galgot** [comment]
Here :
![NextSpace-Mockup01-V3](https://user-images.githubusercontent.com/12213521/60737160-05853600-9f5a-11e9-967d-d68fa732b9f3.jpg)

![NextSpace-Mockup03-V3](https://user-images.githubusercontent.com/12213521/60737177-1170f800-9f5a-11e9-946c-3e173c0f12ae.jpg)

Light now come a bit from left which makes a small highlight on the black plates left bottom edge, NeXT letters are closer to the upper plate.
Gimp 28 file :
[NextSpace-Logo-2000x2000-V.3.xcf.zip](https://github.com/trunkmaster/nextspace/files/3363618/NextSpace-Logo-2000x2000-V.3.xcf.zip)

**Comment by uchuugaka** [comment]
This is really well executed, but a few words of caution.
First, it still uses the name NeXT which Apple still owns, they even still own the related URLs.
Second, and this one kills me after I realized it, I cannot unsee this after I see it, it looks like the Suzuki car logo in glossy black. 

It only looks like Suzuki at larger sizes. Smaller sizes, the spacing is hard to see.
My simple suggestion is to try out what it looks like when you adjust things slightly.
Although we recognize NeXT and macOS app icons as having 3D, realistic and isomorphic qualities, we should remember that the designers also too that isomorphic angle and rotated it a few degrees to be less predictable. Perhaps, try rotating the logo angle similar to the original NeXT logo angle and then from that point, consider mild readjustments of the spacing of the two levels of the object in that rotated isomorphic world.

I might also suggest a similar logo to the original NeXT logo but as though it were like the macOS pkg installers where the box has opened and has something inside, but with the rotated isomorphic design of the original NeXT logo.

**Comment by galgot** [comment]
Fair points. thanks for reminding me, indeed the more I make it looks like the NeXTstep logo, the more it could potentially cause trouble…

As for it looking like the Suzuki S, I don't mind too much. the Suzuki is more a calligraphic asian styled S , not 3 black plates making a S. The general shape resemble yes, but that is why I prefer my first version more: 
The lighting gives more depth and hence one can see more the levels of the "plates", and the idea of progression from the base plate to the upper one passing by the middle one…
If you do it more dark, yes it does looks more like the NeXTstep logo… but then one sees only a S like the Suzuki.
Also the NeXT letters more separated from the plate is better, again the shadows it casks on the plates gives more depth, and it makes it look a bit less like the NeXTstep logo.

I think it shouldn't try to mimic the lightning of the NeXTstep logo.
Because it spoils the idea of the plates… ect… but also because if it looks too much like the NeXTstep, then we could run into copyright problems…
And even here, the "NeXT" should be different.

**Comment by trunkmaster** [comment]
True, we should try to avoid copyright issues. As for Suzuki... I don't care and agree with @galgot.
@galgot, I understand your intentions and concerns. I like the logo idea you've plotted in 1st version. It looks gorgeous at 2000x2000 size. But smaller sizes lost most of the details...

I'm not a graphic designer but I'd like to have a logo that match several criteria:
1. Your idea should be visible and understandable in small sizes (48 for panels, 56 for Dock logo)
2. Image should not mix with the background - sharp clean edges, light edges where light comes from, etc.

I guess version 3 needs more polishing:
- Letters "N" and "X" should not overlap plates interspacing;
- Space between letters and plates' edges should be balanced: now "e" and  "T" look closer to the edges than "N" and "X" (edges that are clearly visible against background).

Several ideas:
- What if you try to place "N" and "X" parallel to the middle plate?
- What if letters' placement will tell the idea of the way from bottom to top? I mean order placement of letters: "N" in place of "X", "e" in place of "N", "X" - "e".

**Comment by galgot** [comment]
Hi Trunkmaster. Sorry for taking much time to answer.
So our two problems were to make it visible and recognizable enough at 56 and 48 sizes, and to avoid possible copyright issues using "NeXT" as on the NeXTstep logo.
While at the same time having something graphically related to NeXT.
Tried discarding the NeXT text, using only the X of "NeXT" with same the four colours, in front of the same black plates. X cause , well it’s a ‘Nix right ? the four colours makes the connection with NeXT Logo. And again the Black plates making a S for Space, while giving a sens of progression…
![NextSpace-Mockup01-V4](https://user-images.githubusercontent.com/12213521/61592627-fa770a80-abd5-11e9-87cc-494eb7679e2d.jpg)
![NextSpace-Mockup02-V4](https://user-images.githubusercontent.com/12213521/61592633-14b0e880-abd6-11e9-9c55-607eff65a2a7.jpg)
![NextSpace-Mockup03-V4](https://user-images.githubusercontent.com/12213521/61592639-272b2200-abd6-11e9-8f70-3efe733f89df.jpg)
I like this one. It's readable at small size and original yet in the same style as the NeXT logo.
Here Gimp 28 file: 
[NextSpace-Logo-2000x2000-V.4.xcf.zip](https://github.com/trunkmaster/nextspace/files/3414711/NextSpace-Logo-2000x2000-V.4.xcf.zip)
You tell me.

**Comment by trunkmaster** [comment]
hi @galgot, my apologies for delay - i've been offline for last 2 weeks.
Thank you for your version 4 of logo - it's indeed fixed copyright issues and much simpler to be clear at different sizes. I appreciate your effort very much.

I think 'X' looks as flat image that was put over 3D 'S' (especially on small sized images). Colors of 'X' looks too vibrant comparing to NeXT's.  And 'X' overlaps shadows of 'S' steps (maybe it's because of too vibrant colors?)...

Frankly speaking, I'd prefer v3 to v4 but copyright issues... I feel confused... Help!

**Comment by alexmyczko** [comment]
why not use the gnustep or gnustep orca logo? https://www.unix-ag.uni-kl.de/~guenther/svgs/gnustep-orca/ ah because it must be called NeXTSPACE?

**Comment by trunkmaster** [comment]
@alexmyczko, maybe because it's GNUstep logo and NEXTSPACE is not GNUstep (NiNG)? ;)

**Comment by galgot** [comment]
Hi Trunkmaster ,
No problem :)  The colors of the "X" are the same as the "NeXT"'s ones. maybe should make the "X" thinner. Will keep tweak this when I have time.
Anyway, I think you want something as good as the NeXT logo, but i'm definitively not Paul Rand :D
Meanwhile , here attached are the Blender (V.2.79) files if you want to play with it.

As for using GNUstep logo, well personally I find it plain ugly. It's the first thing I change when installing WMaker/GNUstep.
[NeXTSpace-Logo-V2.3.4.zip](https://github.com/trunkmaster/nextspace/files/3489663/NeXTSpace-Logo-V2.3.4.zip)

**Comment by alexmyczko** [comment]
But the Orca is cute!!

**Comment by trunkmaster** [comment]
Hi @galgot, thank you very much! I'll play with blender versions when I have time.

**Comment by The-Avatar-of-Virtues** [comment]
Hello. Guys, Ii's nice to greet you all, Mr. Trunkmaster and others. 
And forgive me on my word, which I'm about to say, please.

At first, I'm already using Mr. Galgot's NeXTSpirit Icon Files on Linux, MacOSX, Windows.
I have been combing Mr. Galgot's "Next Folder Icon" with other's Icon, 
then recreating "New NeXT Style Folder Icon" as just for my personal use in amateur way.

So, I have to say, thank you so much, Mr.Galgot.

On the problem of copyright, we can learn from the History of "Afterstep".
See [here](afterstep.org/news.php?show=old).

I know most of people here, like the original Logo made by Keith Ohlfs as the Best.
It's also same as me.

First release of "NeXTSPACE" will try to use the version.2 or 3 logo made by Mr.Galgot  with brave heart.
If the problem occurred on it, just change it like "Afterstep".

Or just add the function for replacing Logos of "Boot splash", "System Branding", "Top of the Dock" etc..
If there was this function, every user can replace their own favorite logo image for these places.

Anyway, you all are now coding.

And If Mr.Galgot devote his all energies to design all part of GUI Shell Parts and Icons in his professional way,
It will make so excited.

Now, It's in the situation of COVID-19.
Take care of yourself, everyone.

**Comment by trunkmaster** [comment]
@The-Avatar-of-Virtues Thanks for sharing your ideas.
I can't check what the history of Afterstep you've mentioned (link doesn't work for me). But as you've may noticed I've not abandoned idea with new logo and set of icons (issue is in opened state). I busy with other things around code now. Anyway I plan to return back to icons/logo issue later.

I hope @galgot hasn't lost interest to this project due to my notes/comments.

**Comment by The-Avatar-of-Virtues** [comment]
Hi, @trunkmaster.
I say sorry for my slow comment.

> I can't check what the history of Afterstep you've mentioned (link doesn't work for me).

I show it again clearly and share some old-school news for you all in this topic.

★The History of Afterstep is below URL:
　http://www.afterstep.org/news.php?show=old

Important quotations will be these from "November 04, 1997" to "November 13, 1997".

★Quotes:
-----------------------------
November 04, 1997
　I have been notified that 
　NeXT Software, Inc. would prefer we not use icons that resemble NEXTSTEP's icons. 
　If you notice any broken images on this homepage, 
　it is because I have decided to remove an image that contains such an icon. 
　Until we create newer and better looking replacement icons to represent our applications, 
　the page will stay this way. 
　If you have an icon you feel would nicely suit AfterStep, 
　please upload it to 
　ftp://oak.sfasu.edu/incoming 
　and we will decide from there 
　what to use with the default distribution which now lacks many critical icons. 
　Thanks!

November 07, 1997
　On the legal front, NeXT's attorney has been very gracious in his dealings with me 
　and has allowed us a period of approximately one month 
　to eliminate the NEXSTEP trademark icons. 
　So, the screenshots are back up for the time being, 
　but I would still like to have those replaced within the week. 
　Let's put those artistic hats on! ;^)

November 13, 1997
　The screenshots have been replaced 
　so hopefully all NeXT trademarks have been eliminated from the page. 
　This is not to say that 
　you should stop producing new icons...keep 'em coming!

---------------------------

That's all!
